source/write/insert.txt

```diff
@@ -0,0 +1,175 @@
+
+The following example inserts two documents into the ``restaurants`` collection:
+
+.. literalinclude:: /includes/write/insert.cpp
+    :language: cpp
+    :dedent:
+    :start-after: start-insert-many
+    :end-before: end-insert-many
+
+Modify Insert Behavior
+----------------------
+
+You can modify the behavior of the ``insert_one()`` and ``insert_many()`` methods by
+passing an instance of the ``mongocxx::v_noabi::options::insert`` class as an optional
+parameter. The following table describes the fields you can set in a
+``mongocxx::v_noabi::options::insert`` instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Field
+     - Description
+
+   * - ``bypass_document_validation``
+     - | If set to ``true``, allows the write to opt out of
+         :manual:`document-level validation </core/schema-validation>`. 
+       | Defaults to ``false``.
+       | **Type**: ``bool``
+
+   * - ``write_concern``
+     - | Sets the write concern for the operation.
+       | Defaults to the write concern of the namespace.
+       | **Type**: ``mongocxx::v_noabi::write_concern``
+
+   * - ``ordered``
+     - | If set to ``true``, the operation stops inserting documents when one insert
+         fails. If ``false``, the operation continues to insert the remaining documents
+         when one insert fails.
+       | Defaults to ``true``.
+       | **Type**: ``bool``
+
+   * - ``comment``
+     - | A comment to attach to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         {+mdb-server+} manual.
+       | **Type**: ``bsoncxx::v_noabi::types::bson_value::view_or_value``
+       
+Example
+~~~~~~~
+
+The following code uses the ``insert_many()`` method to insert three new
+documents into a collection. Because the ``bypass_document_validation`` field
+is set to ``true`` in a ``mongocxx::v_noabi::options::insert`` instance, this
+insert operation bypasses document-level validation:
+
+.. literalinclude:: /includes/write/insert.cpp
+    :language: cpp
+    :dedent:
+    :start-after: start-modify
+    :end-before: end-modify
+
+Additional Information
+----------------------
+
+.. TODO:
+ For runnable code examples of inserting documents with the {+driver-short+}, see
+ :ref:`cpp-write`.
+
+For more examples that use the ``insert_one()`` and ``insert_many()`` methods, see the
+:ref:`cpp-tutorial-insert` section of the Tutorial page.
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+To learn more about any of the methods or types discussed in this
+guide, see the following API documentation:
+
+- `insert_one() <{+api+}/classmongocxx_1_1v__noabi_1_1collection.html#af040f1fcb1ac406037bf1cbcdb300061>`__
+- `insert_many() <{+api+}/classmongocxx_1_1v__noabi_1_1collection.html#a61ebaa0c63296761637ce45115512085>`__
+- `mongocxx::v_noabi::options::insert <{+api+}/classmongocxx_1_1v__noabi_1_1options_1_1insert.html>`__
```

Comment:
Aside: I _would_ recommend links to API reference pages also use root namespace declarations, but Doxygen [doesn't appear to support](https://mongocxx.org/api/mongocxx-3.9.0/classmongocxx_1_1options_1_1insert.html) such a link to a "redirect". 😢 More investigation may be required as part of documentation-related CXX-2745 work (alongside the yet-unresolved Doxygen using-declaration issue).

Reply:
Yeah, removing the v_noabi causes the link to break unfortunately